src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -142,7 +144,10 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
         if (!this.isBaseCompany) return true
         return false
       case EnumUtilities.isTypeAmalgamationOut(this.filing): return true // not supported
-      case EnumUtilities.isTypeAnnualReport(this.filing): return true // not supported
+      case EnumUtilities.isTypeAnnualReport(this.filing):
+        // enable AR corrections for specified legal types only
+        if (GetFeatureFlag('supported-ar-correction-entities').includes(this.getLegalType)) return false
+        return true
```

Comment:
If the feature flag include the current legal type (eg, "CP") then don't disable the "File a Correction" menu item.

Reply:
Screenshot (from Dev):

![image](https://github.com/user-attachments/assets/38f3d5d1-0aed-46d8-9e51-2407e613e9ab)